src/assets/css/app.css

```diff
@@ -0,0 +1,63 @@
+.App {
+  overflow: hidden;
+}
+
+.all-caps {
+  text-transform: uppercase;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+}
+
+.backBG {
+  padding: 8em 4em;
+}
+
+.container {
+  max-width: 1400px;
+  padding-left: 1em;
+  padding-right: 1em;
+}
+
+.bg-dark-grey {
+  background-color: #f3f4f8;
+}
+
+.center {
+  margin: auto;
+}
+
+.max-width-90 {
+  max-width: 90%;
+}
+
+.flex-row {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+}
+
+.flex-row * {
+  margin: 5px 5px 5px 5px;
+  padding: 5px 0 5px 0;
+}
+
+/* Small screens less than 720px */
+@media screen and (max-width: 720px) {
+  .flex-row * {
+    margin: 5px 0;
+  }
+
+  /* .max-width-90 { width: 99%; margin: auto; }
```

Comment:
> * [ ]  It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .hintrc, .eslintrc.json etc).

- Thank you @wandji20 But clearly the previews reviewer fail to install the dependencies claiming my app is not running, I decided to add the missing files to enable them nit have issues with their computer, now you requested I change alright thanks for the heads up I will make the required changes